scripts/integration/Snakefile

```diff
@@ -16,52 +15,52 @@ rule integration_run_python:
         command: {params.cmd}
         hvgs: {params.hvgs}
         cell type option: {params.cell_type}
+        output: {output[0]}
         """
     params:
         batch_key = lambda wildcards: cfg.get_from_scenario(wildcards.scenario, key="batch_key"),
         cell_type = cfg.get_celltype_option_for_integration,
-        hvgs      = lambda wildcards, input: cfg.get_hvg(wildcards, input.adata[0]),
+        hvgs      = cfg.get_hvg,
```

Comment:
you don't need an input argument here?

Reply:
no, Snakemake is smart :D 

This notation works when the only input is `wildcards`. I removed the input file pattern, as the python script doesn't require any separate HVG input (it's computed in the script).

Reply:
cool :). Okay!